facebook/facebook.c

```diff
@@ -142,14 +142,17 @@ fb_cb_api_connect(FbApi *api, gpointer data)
     account_t *acct;
     FbData *fata = data;
     struct im_connection *ic;
+    gint show_unread;
 
     ic = fb_data_get_connection(fata);
     acct = ic->acc;
 
+    show_unread = !(ic->flags & OPT_LOGGED_IN);
```

Comment:
The name of this variable doesn't really describe it properly. I'm not sure why you need a variable for this either